packages/components/src/components/ConfirmationModal/ConfirmationModal.test.tsx

```diff
@@ -0,0 +1,88 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import { useModalState } from "../Modal/index";
+import { ConfirmationModal } from "./ConfirmationModal";
+
+const OPEN_MODAL_TEXT = "Open modal";
+
+const onConfirm = jest.fn();
+
+const ExampleModal = () => {
+  const state = useModalState();
+
+  return (
+    <div>
+      <button onClick={state.show}>{OPEN_MODAL_TEXT}</button>
+
+      <ConfirmationModal onConfirm={onConfirm} state={state}>
+        Do you want to confirm?
+      </ConfirmationModal>
+    </div>
+  );
+};
+
+const CustomModal = () => {
+  const state = useModalState();
+
+  return (
+    <div>
+      <button onClick={state.show}>{OPEN_MODAL_TEXT}</button>
+
+      <ConfirmationModal
+        buttonLabel="Delete"
+        buttonVariant="destructive"
```

Comment:
Shouldn't it be a ConfirmationDialog variant instead of just a button variant? Do we have a definition from design side?